packages/spor-react/src/loader/ContentLoader.tsx

```diff
@@ -1,20 +1,31 @@
 import { Box, BoxProps } from "@chakra-ui/react";
-import { contentLoaderData } from "@vygruppen/spor-loader";
+import {
+  contentLoaderData,
+  contentLoaderPrideData,
+} from "@vygruppen/spor-loader";
 import React from "react";
 import { ClientOnly } from "./ClientOnly";
 import Lottie from "./Lottie";
+import { usePride } from "../pride/PrideProvider";
 
 export type ContentLoaderProps = BoxProps;
 /**
  * ContentLoader is a component that renders a loading animation.
  * It should mostly be used for
  */
 export const ContentLoader = ({ children, ...props }: ContentLoaderProps) => {
+  const { isPride } = usePride();
   return (
     <Box {...props}>
       <Box maxWidth="140px" marginX="auto">
         <ClientOnly>
-          {() => <Lottie animationData={contentLoaderData} />}
+          {() => (
```

Comment:
Er ikke så kjent med Lottie, men hvorfor wrappes den i en funksjon?

Reply:
Lottie trenger ikke at det wrappes i en funksjon generelt sett.

Reply:
https://nalu.wiki/nalu/p/fixing-dependency-errors-in-remix-187#client-only-work-arounds

Tror jeg fant svaret her ja :-)